### PR TITLE
[3sat] Download the rtmp URL for the meta file

### LINKED ISSUE #7680: downloads only meta file on 3sat Mediathek
It only downloads some meta file with an RTMP URL in it:

    mcnesium@mcobst:~/Downloads$ youtube-dl -v "http://www.3sat.de/mediathek/?obj=55586"
    [debug] System config: []
    [debug] User config: []
    [debug] Command-line args: [u'-v', u'http://www.3sat.de/mediathek/?obj=55586']
    [debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
    [debug] youtube-dl version 2015.11.24
    [debug] Python version 2.7.10 - Darwin-15.0.0-x86_64-i386-64bit
    [debug] exe versions: ffmpeg 2.8.2, ffprobe 2.8.2, rtmpdump 2.4
    [debug] Proxy map: {}
    [3sat] 55586: Downloading video details
    [debug] Invoking downloader on u'http://fstreaming.zdf.de/3sat/hd/15/11/151127_wohlstand_ard.meta'
    [download] Destination: Wohlstand für alle-55586.meta
    [download] 100% of 172.00B in 00:00
    mcnesium@mcobst:~/Downloads$ ll -h "Wohlstand für alle-55586.meta"
    -rw-r--r-- 1 mcnesium staff 172 Nov 23 15:39 Wohlstand für alle-55586.meta
    mcnesium@mcobst:~/Downloads$ cat Wohlstand\ für\ alle-55586.meta
    <metafile version="1.0">
      <default-stream-url>rtmp://cp125301.edgefcs.net/ondemand/mp4:3sat/15/11/151127_wohlstand_ard_3256k_p15v12.mp4</default-stream-url>
    </metafile>

### PR DESCRIPTION
**Pull request by akirk**
Fixes #7680 and #8104

### CODE REVIEW

**Comment by remitamine** [comment]
a better way to solve this is to reuse the code from ZDFIE like in PhoenixIE(this error already fixed in ZDFIE https://github.com/rg3/youtube-dl/commit/a5c1d95500898541cd6b19b32963883db421c4b4).

**Comment by akirk** [comment]
@remitamine ok, that was easier than I expected. I modified this PR accordingly.

**Comment by akirk** [comment]
Ok, I spoke too early. It works when you manually specify the HLS download format, but the RTMP format is selected best and fails with
```
ERROR: unable to download video data: <urlopen error unknown url type: rtmpt>
```

Thus, I have reverted my PR back to my original approach, feel free to improve it.

File: youtube_dl/extractor/dreisat.py [hunk 1]
```diff
@@ -34,54 +28,6 @@ class DreiSatIE(InfoExtractor):
     def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         video_id = mobj.group('id')
-        details_url = 'http://www.3sat.de/mediathek/xmlservice/web/beitragsDetails?ak=web&id=%s' % video_id
-        details_doc = self._download_xml(details_url, video_id, 'Downloading video details')
-
-        status_code = details_doc.find('./status/statuscode')
```
Comment (remitamine) [inline youtube_dl/extractor/dreisat.py]: this part is not handled by ZDFIE, i will move it to extract_from_xml_url in zdf.py.

Comment (akirk) [inline youtube_dl/extractor/dreisat.py]: Great, thanks!

**Comment by remitamine** [comment]
> ERROR: unable to download video data: <urlopen error unknown url type: rtmpt>

thanks for testing,i didn't notice the problem with rtmpt format before.i fixed this problem in https://github.com/rg3/youtube-dl/commit/6b461026616b0e3ede9a302c74fc437541e19343.
rtmp formats already extracted using the smil manifest so there is no need to extract them from .meta.

**Pull request closed without merging**